ipie/analysis/extraction.py

```diff
@@ -25,12 +25,17 @@
 from ipie.utils.misc import get_from_dict
 
 
-def extract_hdf5_data(filename, block_idx=1):
+def extract_hdf5_data(filename, block_idx=1, complexQ=False):
```

Comment:
Make this a new function specific for free projection rather than adding logic to the function.